.github/workflows/build.yml

```
@@ -0,0 +1,61 @@
+name: Electron Build
+
+on: [push]
```

このまま修正コミット入れると、ガンガン走ってしまうと思うので、一旦masterへのpushにしてもらえると！

```suggestion
on:
  push:
    branches: [master]
```